load.php

```diff
@@ -261,6 +261,41 @@ function perflab_maybe_remove_object_cache_dropin() {
 }
 register_deactivation_hook( __FILE__, 'perflab_maybe_remove_object_cache_dropin' );
 
+/**
+ * Redirects module pages to the peformance feature page.
+ *
+ * @since n.e.x.t
+ *
+ * @global $plugin_page
+ */
+function perflab_no_access_redirect_module_to_peformance_feature_page() {
+	global $plugin_page;
+
+	if ( 'perflab-modules' !== $plugin_page ) {
+		return;
+	}
+
+	if ( current_user_can( 'manage_options' ) ) {
+		wp_safe_redirect(
+			add_query_arg( 'page', PERFLAB_SCREEN )
+		);
+		exit;
+	}
+}
+add_action( 'admin_page_access_denied', 'perflab_no_access_redirect_module_to_peformance_feature_page' );
+
+/**
+ * Cleanup function to delete 'perflab_modules_settings' option if present.
+ *
+ * @since n.e.x.t
+ */
+function perflab_cleanup_option() {
+	if ( current_user_can( 'manage_options' ) && get_option( 'perflab_modules_settings' ) ) {
```

Comment:
Do we need the `get_option()` check? I think `delete_option()` handles it on its own.

Reply:
Remove it in https://github.com/WordPress/performance/pull/1060/commits/959377cec9dddc5d8997b7bb5e9bd03e49868f90